cardano-node-chairman/test/Spec/Chairman/Cardano.hs

```diff
@@ -4,21 +4,20 @@
 
 module Spec.Chairman.Cardano where
 
-import           Cardano.Testnet (NodeRuntime (nodeName), allNodes,
-                   cardanoTestnetDefault, mkConf)
+import           Cardano.Testnet (allNodes, cardanoTestnetDefault, mkConf)
 
 import           Data.Default.Class
+
 import           Testnet.Property.Util (integrationRetryWorkspace)
 
 import qualified Hedgehog as H
 
 import           Spec.Chairman.Chairman (chairmanOver)
 
--- TODO: Conway broken in conway
 hprop_chairman :: H.Property
 hprop_chairman = integrationRetryWorkspace 2 "cardano-chairman" $ \tempAbsPath' -> do
   conf <- mkConf tempAbsPath'
 
-  allNodes' <- fmap nodeName . allNodes <$> cardanoTestnetDefault def def conf
+  allNodes' <- allNodes <$> cardanoTestnetDefault def def conf
 
-  chairmanOver 120 50 conf allNodes'
+  chairmanOver 120 40 conf allNodes'
```

Comment:
Why is the required progress reduced?

Reply:
It was progressing to just 48 block sometimes on my local machine, I have not seen this in CI. I suspect that's because of more frequent rollbacks in 9.2. I've reverted height requirement to 50.

Let's revisit this later once chairman starts to get flaky.